src/utils/countdown.js

```diff
@@ -0,0 +1,30 @@
+const EventEmitter = require('events');
```

Comment:
Please investigate how this behave in the browser.It's nodejs native module. How much it costs. How big is the file. Wouldn't solutions mentioned in Vue3 migration guide a better option: https://v3.vuejs.org/guide/migration/events-api.html#migration-strategy.